ydb/docs/en/core/reference/ydb-cli/sql.md

```diff
@@ -0,0 +1,120 @@
+# Query execution
+
+You can use the `{{ ydb-cli }} sql` subcommand to execute any SQL query. The query can be of any type (DDL, DML e.t.c.) and can consist of several sub-queries. The `{{ ydb-cli }} sql` subcommand establishes a streaming connection and retrieves data through it. With the in-stream query execution, no limit is imposed on the amount of data read.
+
+General format of the command:
+
+```bash
+{{ ydb-cli }} [global options...] sql [options...]
+```
+
+* `global options`: [Global parameters](commands/global-options.md).
+* `options`: [Parameters of the subcommand](#options).
+
+View the description of this command by calling it with `--help` option:
+
+```bash
+{{ ydb-cli }} sql --help
+```
+
+## Parameters of the subcommand {#options}
+
+#|
+|| Name | Description ||
+|| `-h`, `--help` | Print general usage help ||
+|| `-hh` | Print complete usage help. Prints some specific options that are not printed with `--help` ||
+|| `-s`, `--script` | Script (query) text to execute. ||
+|| `-f`, `--file` | Path to a file with query text to execute. Path `-` means reading query text from `stdin`. ||
```

Comment:
I'd highlight that `-` is probably incompatible with passing parameters via `stdin`.